src/Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -3,14 +3,33 @@ namespace NServiceBus
 {
     public class static NewtonsoftConfigurationExtensions
     {
+        [System.ObsoleteAttribute("Use `NewtonsoftJsonSerializer` instead. Will be treated as an error from version " +
```

Comment:
I'm not sure if these obsoletion messages carry enough context for a user to know what change to make. The message used on the master branch is probably better.

Reply:
I have updated to use the same messages as on master.